### DISCUSSION TITLE
How to remove cockpit-files "app?"

### DISCUSSION THREAD

**Discussion by HVR88** [cockpit-project/cockpit-files#626] [Q&A]
**On Fedora Server 40**

Files doesn't appear in Cockpit in the Applications list after being installed (no appstream info?)

_Files doesn't appear in list of installed packages on the system and therefore can't be removed with_ `dnf remove cockpit-files`

**_How can cockpit files be removed from the system and Cockpit easily/cleanly?_**

**Comment by jelly** [comment]
`dnf remove cockpit-files` should do the trick.

**Reply by HVR88** [reply]
I should have mentioned that I tried the obvious way of removing a named package.   It doesn't work because no package with the name "cockpit-files" can be found.

**Reply by martinpitt** [reply]
@HVR88 Check `cockpit-bridge --packages`? It may be a local `sudo make install` installation, or perhaps even in your ~/.local/share/cockpit.
